dev/docker/oc10.config.php

```diff
@@ -2,5 +2,6 @@
 $CONFIG = [
     'web.baseUrl' => 'http://host.docker.internal:8080/index.php/apps/web',
     'web.rewriteLinks' => true,
+    'sharing.showPublicLinkQuickAction' => true,
```

Comment:
dev leftover or intended change to our dev setup?

Reply:
Had the same questions, was reluctant to commit it but @fschade went ahead. I get the reasoning though and would leave it in our dev setup since otherwise we'd fall back to internal pings about "how to get it work" which is suboptimal

Reply:
i would keep it, this enables the same quicklink behavior for oc10 as in ocis...